mars/dataframe/base/cut.py

```diff
@@ -362,14 +369,25 @@ def execute(cls, ctx, op):
         bins = ctx[op.bins.key] if isinstance(op.bins, ENTITY_TYPE) else op.bins
         labels = ctx[op.labels.key] if isinstance(op.labels, ENTITY_TYPE) else op.labels
 
-        cut = partial(
-            pd.cut,
-            right=op.right,
-            retbins=op.retbins,
-            precision=op.precision,
-            include_lowest=op.include_lowest,
-            duplicates=op.duplicates,
-        )
+        if pd.__version__ >= "1.1.0":
```

Comment:
Better to use parse_version method in `mars.lib.version`.

Reply:
Good suggestion. I will fix it.